site/src/modules/provisioners/useCompatibleProvisioners.ts

```diff
@@ -0,0 +1,24 @@
+import { ProvisionerDaemon } from "api/typesGenerated";
+
+export const provisionersUnhealthy = (provisioners : ProvisionerDaemon[]) => {
+	return provisioners.reduce((allUnhealthy, provisioner) => {
+		if (!allUnhealthy) {
+			// If we've found one healthy provisioner, then we don't need to look at the rest
+			return allUnhealthy;
+		}
+		// Otherwise, all provisioners so far have been unhealthy, so we check the next one
+
+		// If a provisioner has no last_seen_at value, then it's considered unhealthy
+		if (!provisioner.last_seen_at) {
+			return allUnhealthy;
+		}
+
+		// If a provisioner has not been seen within the last 60 seconds, then it's considered unhealthy
+		const lastSeen = new Date(provisioner.last_seen_at);
+		const oneMinuteAgo = new Date(Date.now() - 60000);
```

Comment:
I would recommend using `dayjs().subtract(1, "minute")` instead here

Reply:
this function is defunct and has been removed. but thanks for the tip